util/get-toolchain.py

```diff
@@ -138,6 +144,16 @@ def main():
                         required=False,
                         default=TARGET_DIR,
                         help="Target directory (default: %(default)s)")
+    parser.add_argument('--release-dir',
+                        required=False,
+                        help="""Release directory, if different from target
+                        directory. If not set, this defaults to the target
+                        directory. Useful in corporate environments where a
+                        more global tool installation area is preferred, but
+                        due to IT restrictions, the installation has to be done
+                        in a temporary staging area first. The tool version
+                        to be installed is appended to this path by default.
+                        """)
```

Comment:
I find the naming and explanation rather confusing now. Can we do two things:

- Switch to a more common naming, such as
  * `install-dir`: the final installation path
  * `dest-dir` (or `install-staging-dir`): the installation staging directory, from where it is moved to the final install-dir.

  See https://www.gnu.org/software/automake/manual/html_node/Staged-Installs.html for the automake documentation on it.
* Explain what these directories do not by example. You can also remove the "corporate IT environment" motivation from the help text.